### ISSUE TITLE
Server forced to stop by watchdog due to tick delay

### ISSUE THREAD

**Issue by RickyvdBerg** [AllTheMods/ATM-6#368]
**Describe the bug**
Server lags behind, watchdog force stops the server.
I'm running a server with 6GB RAM and it doesn't look like it's needing anymore.  
However, the server still seems to hang a lot, which causes the watchdog to force stop the server.  
It may just be that it's just the server performance however, it only started happening after going to the nether.

I feel like it's not related to the server specs. I glanced over the crash report myself but I couldn't find anything obvious.  

[crash-2020-11-23_18.33.23-server.txt](https://github.com/AllTheMods/ATM-6/files/5585097/crash-2020-11-23_18.33.23-server.txt)

**Comment by RickyvdBerg** [comment]
spark profile during a big hangup: https://spark.lucko.me/#uptL8awdBc

**Comment by ViRb3** [comment]
Looks like chunk loading is taking a very long time, namely, writing the NBT data. Do you have any block or item with an extreme amount of NBT data? Or is the I/O (disk speed) very slow on your server?

**Comment by RickyvdBerg** [comment]
The only block or item would be my weapon with all the SGear stuff or the dank null. It still happens when they aren't on me. Also given the fact that it's on chunks loading, when I recorded that profile it was when generating new chunks in the nether. So correct me if I'm wrong but that would eliminate any player bases etc. The server is also very new (2 days old) so I assume it's not due to any structures build by players (only 3 players on the server).   
  
I also just checked the server hardware and if I were to believe the server host hardware specs it should run on an m2 SSD with 3.4gb/s reads and 1.6gb writes transfer speeds.
